### PR TITLE
[PDMO-33] change presentation of deleted and preferred from checkboxes

### PR DESCRIPTION
**Pull request by Ewanjiru**
# JIRA TICKET NAME:

PDMO-33  [Change presentation for deleted and preferred from checkboxes](https://issues.openmrs.org/browse/PDMO-33)

# SUMMARY:

Currently details that are deleted or preferred are shown using a checkbox. This PR changes that sort of presentation to badges for preferred and a 'soft deleted' watermark text for the forms showing deleted details such as encounters and strikethrough for the listed details under visits.

### CODE REVIEW

File: app/js/components/identifiers/identifiers.jsx [hunk 1]
```diff
@@ -249,14 +249,23 @@ export default class Identifiers extends React.Component {
...
+
+                    <div className="viewCard">
+                      {id.identifierType.display}{': '}
+                      {this.state.activeCard !== index ? id.identifier : editIdentifiers.identifier || id.identifier}
+                      {'   '}
+                      {this.state.editState === false &&
+                        <a onClick={() => this.callEdit(index, id.uuid)}>Edit
+											</a>
+                      }
+                    </div>
```
Comment (jacksono) [inline app/js/components/identifiers/identifiers.jsx]: The closing anchor tag seems to be indented too much

File: app/js/components/names.jsx [hunk 2]
```diff
@@ -121,140 +124,125 @@ class Name extends React.Component {
...
+              <input className='form-control'
+                name='givenName'
+                type='text'
+                value={this.state.givenName}
+                onChange={this.handleChange}
+                readOnly={
+                  this.state.display === 'view' ?
+                    'readonly' : null
+                }
+              />
```
Comment (jacksono) [inline app/js/components/names.jsx:161]: Indentation can be improved for the readonly property

File: app/js/components/names.jsx [hunk 3]
```diff
@@ -121,140 +124,125 @@ class Name extends React.Component {
...
+              <input className='form-control'
+                name='middleName'
+                type='text'
+                value={this.state.middleName}
+                onChange={this.handleChange}
+                readOnly={
+                  this.state.display === 'view' ?
+                    'readonly' : null
+                }
+              />
```
Comment (jacksono) [inline app/js/components/names.jsx:184]: Improve indentation here

File: app/js/components/names.jsx [hunk 4]
```diff
@@ -121,140 +124,125 @@ class Name extends React.Component {
...
+              <input className='form-control'
+                name='familyName'
+                type='text'
+                value={this.state.familyName}
+                onChange={this.handleChange}
+                readOnly={
+                  this.state.display === 'view' ?
+                    'readonly' : null
+                }
+              />
```
Comment (jacksono) [inline app/js/components/names.jsx:200]: Ditto

File: app/js/components/demographics.jsx [hunk 5]
```diff
@@ -121,147 +106,124 @@ export default class Demographics extends React.Component {
...
+
+          <div className="form-group ">
+            <label className="control-label col-sm-2">Birth Date</label>
+            <div className="col-sm-5">
+              <DatePicker
+                dateFormat="DD-MM-YYYY"
+                className="form-control"
+                name="birthdate"
+                value={this.state.birthdate} onChange={this.handleTimeChange('birthdate')} />
+            </div>
```
Comment (jacksono) [inline app/js/components/demographics.jsx]: The onchange property could go on a new line to be consistent with the previous ones

**Review by jacksono — CHANGES_REQUESTED** [review]
Some comments on indentation

File: app/js/components/demographics.jsx (hunk 5 above)
Comment (Ewanjiru) [reply app/js/components/demographics.jsx]: @jacksono  well noted. Working on above.

File: app/js/components/names.jsx (hunk 2 above)
Comment (Ewanjiru) [reply app/js/components/names.jsx:161]: corrected.

File: app/js/components/identifiers/identifiers.jsx (hunk 1 above)
Comment (Ewanjiru) [reply app/js/components/identifiers/identifiers.jsx]: corrected

File: app/js/components/names.jsx (hunk 4 above)
Comment (Ewanjiru) [reply app/js/components/names.jsx:200]: Done

**Comment by dkayiwa** [comment]
@Ewanjiru can you look into the merge conflicts on this?

**Comment by Ewanjiru** [comment]
@dkayiwa  working on them.

**Comment by Ewanjiru** [comment]
@dkayiwa I have resolved the merge conflicts.

**Comment by dkayiwa** [comment]
You need a second reviewer.

**Comment by Ewanjiru** [comment]
Yes. I have requested for another review.

File: app/css/obsadmin.css [hunk 6]
```diff
@@ -220,7 +219,9 @@ li>a:focus {
 .collapsible-header-top {
     margin-top: 20px;
 }
-
+#deletePatient{
```
Comment (wanjikum) [inline app/css/obsadmin.css]: Please use #delete-patient for consistency

File: app/css/obsadmin.css [hunk 7]
```diff
@@ -220,7 +219,9 @@ li>a:focus {
 .collapsible-header-top {
     margin-top: 20px;
 }
-
```
Comment (wanjikum) [inline app/css/obsadmin.css]: why did you remove the space?

File: app/css/obsadmin.css [hunk 8]
```diff
@@ -265,7 +266,13 @@ li>a:focus {
     background-color: #f7f7f9;
     border-bottom: 1px solid rgba(0,0,0,.125);
 }
-
```
Comment (wanjikum) [inline app/css/obsadmin.css]: Why did you remove the space?

File: app/css/obsadmin.css [hunk 9]
```diff
@@ -265,7 +266,13 @@ li>a:focus {
...
+.preffered{
+    display: block;
+    margin-left: 2%;
+}
+.badge-info {
```
Comment (wanjikum) [inline app/css/obsadmin.css]: Please add a space before this

File: app/css/obsadmin.css [hunk 10]
```diff
@@ -265,7 +266,13 @@ li>a:focus {
...
+  background-color: #5cb85c;
+}
```
Comment (wanjikum) [inline app/css/obsadmin.css]: Space please

Comment (Ewanjiru) [reply app/css/obsadmin.css]: @wanjikum  Working on it.

Comment (wanjikum) [reply app/css/obsadmin.css]: Alright 👍

Comment (Ewanjiru) [reply app/css/obsadmin.css]: All resolved. Thanks
